pkg/kotsutil/kots.go

```diff
@@ -396,6 +406,90 @@ func (o KotsKinds) Marshal(g string, v string, k string) (string, error) {
 	return "", errors.Errorf("unknown gvk %s/%s, Kind=%s", g, v, k)
 }
 
+func (k *KotsKinds) addKotsKind(content []byte) error {
```

Comment:
plural would be more accurate
```suggestion
func (k *KotsKinds) addKotsKinds(content []byte) error {
```